plugins/web/ui/src/routes/status/+page.svelte

```diff
@@ -20,7 +20,7 @@
       // under the current BASE_URL of the frontend deployment
       const API_BASE =
         import.meta.env.VITE_VAST_API_ENDPOINT ?? `${import.meta.env.BASE_URL}api/v0`;
-      const url = `${API_BASE}/status?verbosity=detailed`;
```

Comment:
I tried to build the static binary via Nix and run it - looks like I am getting a 404 on http://localhost:42001. Has anything changed about how the frontend built files get embedded?

Reply:
We didn't change anything in that regard. Does this work for you?

```
docker run -p 42001:42001 ghcr.io/tenzir/vast:story-sc-40408-schema-stats-in-catalog start
```

Otherwise, let's hop on a quick call to debug.

Reply:
I'm assuming that this is okay; if you still have issues it's independent of this PR anyways, so I'm closing this.